File: Harvard-CS50x/pset6/cash/cash.py
from cs50 import get_float

#Request user input
dollars = 0

while (dollars <= 0 ):
    dollars = get_float("change owed: ")

coins = 0
cents = round(100 * dollars, 0)

while (cents > 0):
    if (cents >= 25):
        cents -= 25
        coins += 1
    elif (cents >= 10):
        cents -= 10
        coins += 1
    elif (cents >= 5):
        cents -= 5
        coins += 1
    else:
        cents -= 1
        coins += 1
print(f"Change Owed: {coins}")

File: Harvard-CS50x/pset1/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float change;
    do
    {
        change = get_float("Change owed: ");
    }      
    while (change < 0);
    
    //convert change to int
    int coins = round(change * 100);
    
    //find out how many of each coin
    int counter = 0;
    while (coins > 24)
    {
        coins = coins - 25;
        counter ++;
    }
    while (coins > 9)
    {
        coins = coins - 10;
        counter ++;
    }
    while (coins > 4)
    {
        coins = coins - 5;
        counter ++;
    }
    counter = counter + coins;
    printf("%i\n", counter);
}
